recipes/ipycytoscape/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "ipycytoscape" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 95e41ec5c9d3e5633a5364eee891b3a0158338a6203d7d89a3d5768df61d78c6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py2k]
```

Could this recipe be ``noarch: python`` please? and you can return the python constrains?
```suggestion
  noarch: python
```